docs/api-reference/classes/ImageDiffError.mdx

```diff
@@ -0,0 +1,146 @@
+[testplane](../README.mdx) / ImageDiffError
+
+# Class: ImageDiffError
+
+## Extends
+
+- `BaseStateError`
+
+## Constructors
+
+### new ImageDiffError()
+
+```ts
+new ImageDiffError(__namedParameters): ImageDiffError
+```
+
+#### Parameters
+
+| Parameter | Type |
+| ------ | ------ |
+| `__namedParameters` | `object` |
+| `__namedParameters.currImg` | [`ImageInfo`](../interfaces/ImageInfo.mdx) |
+| `__namedParameters.diffAreas`? | `DiffAreas` |
+| `__namedParameters.diffBuffer` | `Buffer` |
+| `__namedParameters.diffOpts` | `DiffOptions` |
+| `__namedParameters.diffRatio` | `number` |
+| `__namedParameters.differentPixels` | `number` |
+| `__namedParameters.refImg` | [`RefImageInfo`](../interfaces/RefImageInfo.mdx) |
+| `__namedParameters.stateName` | `string` |
+
+#### Returns
+
+[`ImageDiffError`](ImageDiffError.mdx)
+
+#### Overrides
+
+`BaseStateError.constructor`
+
+#### Defined in
+
+[src/browser/commands/assert-view/errors/image-diff-error.ts:79](https://github.com/gemini-testing/testplane/blob/57d0b182c37e21d8bdf52acbb137a028841767af/src/browser/commands/assert-view/errors/image-diff-error.ts#L79)
+
+## Properties
+
+| Property | Modifier | Type | Overrides | Inherited from | Defined in |
+| ------ | ------ | ------ | ------ | ------ | ------ |
+| `currImg` | `public` | [`ImageInfo`](../interfaces/ImageInfo.mdx) | - | `BaseStateError.currImg` | [src/browser/commands/assert-view/errors/base-state-error.ts:4](https://github.com/gemini-testing/testplane/blob/57d0b182c37e21d8bdf52acbb137a028841767af/src/browser/commands/assert-view/errors/base-state-error.ts#L4) |
+| `diffBounds?` | `public` | `CoordBounds` | - | - | [src/browser/commands/assert-view/errors/image-diff-error.ts:43](https://github.com/gemini-testing/testplane/blob/57d0b182c37e21d8bdf52acbb137a028841767af/src/browser/commands/assert-view/errors/image-diff-error.ts#L43) |
+| `diffBuffer` | `public` | `Buffer` | - | - | [src/browser/commands/assert-view/errors/image-diff-error.ts:45](https://github.com/gemini-testing/testplane/blob/57d0b182c37e21d8bdf52acbb137a028841767af/src/browser/commands/assert-view/errors/image-diff-error.ts#L45) |
+| `diffClusters?` | `public` | `CoordBounds`[] | - | - | [src/browser/commands/assert-view/errors/image-diff-error.ts:44](https://github.com/gemini-testing/testplane/blob/57d0b182c37e21d8bdf52acbb137a028841767af/src/browser/commands/assert-view/errors/image-diff-error.ts#L44) |
```

Comment:
Types are now formatted as `string`|`string`[], is it possible to make them `string | string[]` ?

Makes them hard to read, IMO:
![](https://i.imgur.com/EWVYBc7.png)